### DISCUSSION TITLE
Add view mode to Settings

### DISCUSSION THREAD

**Discussion by emfhal** [langfuse/langfuse#2358] [Ideas]
Tree/Timeline (/slides/cards)

**Comment by marcklingen** [comment]
can you specify? would you like to set a default in the settings?

**Reply by emfhal** [reply]
I have some suggestions regarding the default view mode behavior and potential additional options.  

One approach could be to store the last chosen view mode as a setting. This would mean that whenever a user switches between traces, the previously chosen mode (either timeline or tree) would be applied by default.  

It would also be beneficial to provide a clear option for users to switch between view modes. 
This could be a toggle button or a menu option within the user interface, allowing users to easily switch between tree and timeline views, regardless of the default setting.  Furthermore, it might be interesting to explore the possibility of introducing new view modes, such as cards or slides, to cater to different data visualization preferences.  

Lastly, for the timeline view, it could be more efficient to allow users to open models directly by clicking on a row. 
Alternatively, keyboard shortcuts like 'Enter' or 'Space' could be used.  In summary, this comment discusses various approaches to managing view mode preferences and introduces ideas for new functionalities. These include additional view modes and improved interaction within the timeline view.

**Reply by marcklingen** [reply]
thanks for sharing! cc @marliessophie who has built the new Timeline view
